webviewer/demo/giowebview/demo.go

```diff
@@ -91,7 +91,9 @@ func main() {
 				os.Exit(0)
 				return
 			case app.FrameEvent:
-				plugin.Install(window, evt)
+				once.Do(func() {
+					plugin.Install(window, evt)
+				})
```

Comment:
Maybe `Install` was not the correct naming, `Hijacking` would be better. The `Install` replaces the current `app.FrameEvent`, using `unsafe` hacks, so it must run on each frame: https://github.com/gioui-plugins/gio-plugins/blob/main/plugin/plugin.go#L155

Reply:
So I need you to change further, I don't understand the details very well, at the moment all I can do is get all the demos running, the webview is a bit problematic and stuck. The remaining details need to be further implemented by you, and hopefully this will save you time and effort. Also, the webview doesn't seem to have the interface for disabling devtool under Windows, the dark theme if you need it, I've done testing on my side, and the drag and drop files haven't worked out yet

Reply:
For me,I want to implement a function of transferring files between Android and PC,This needs to turn on the function of mobile phone hotspot or PC hotspot,Do you have any suggestions for this?Oh yes,It's Bluetooth open request + wifi open request,As for the hotspot establishment process is a mystery